colorbleed/fusion/__init__.py

```diff
@@ -20,6 +22,8 @@ def install():
 
     pyblish.register_callback("instanceToggled", on_pyblish_instance_toggled)
 
+    colorbleed.register_manager_actions()
```

Comment:
No need to take this from the colorbleed global space. The fusion actions should just belong to the fusion host - install it like any of the other actions.

Maybe it should even be added as files in:

```plugins/fusion/inventory```

:+1:

Reply:
Nice! I like that idea :)